back/src/auth/auth.controller.ts

```diff
@@ -66,6 +69,7 @@ export class AuthController {
 	async disableTotp(@Req() req: Request, @Res() res: Response) {
 		const user = await this.userService.disableTotp(req.user.id);
 		const token: { access_token: string } = await this.authService.validateUser(user);
+		await this.deviceService.deleteAllDevices(req.user.id);
```

Comment:
Could you please make a function for this so we could "untrust all other websites" ?

It's only because I feel like it's a quick addition that I'm asking, knowing that it wouldn't take too long...

I'll let you be judge, tell me if you want me to create an "out of project scope" issue that we could do later if we have spare time.